missing depends: traverse

```
% textlint test.org

✖ Error
Cannot find module 'traverse'
Require stack:
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/org-to-ast.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/OrgProcessor.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/index.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-module-loader.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-engine-core.js
- ~/nodejs/lib/node_modules/textlint/lib/src/textlint-engine.js
- ~/nodejs/lib/node_modules/textlint/lib/src/cli.js
- ~/nodejs/lib/node_modules/textlint/bin/textlint.js

✖ Stack trace
Error: Cannot find module 'traverse'
Require stack:
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/org-to-ast.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/OrgProcessor.js
- ~/nodejs/lib/node_modules/textlint-plugin-org/lib/index.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-module-loader.js
- ~/nodejs/lib/node_modules/textlint/lib/src/engine/textlint-engine-core.js
- ~/nodejs/lib/node_modules/textlint/lib/src/textlint-engine.js
- ~/nodejs/lib/node_modules/textlint/lib/src/cli.js
- ~/nodejs/lib/node_modules/textlint/bin/textlint.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (~/node_modules/textlint-plugin-org/lib/org-to-ast.js:8:34)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
```

Best Wishes,

--- a/package.json
+++ b/package.json
@@ -30,7 +30,8 @@
     "dependencies": {
         "@types/date-fns": "^2.6.0",
         "orga": "2.4.9",
-        "structured-source": "^3.0.2"
+        "structured-source": "^3.0.2",
+        "traverse": "^0.6.6"
     },
     "devDependencies": {
         "@textlint/ast-tester": "^12.0.0",
@@ -45,7 +46,6 @@
         "mocha": "^10.0.0",
         "power-assert": "^1.6.1",
         "textlint-rule-max-comma": "^2.0.2",
-        "traverse": "^0.6.6",
         "ts-node": "^10.0.0",
         "ts-node-test-register": "^10.0.0",
         "typescript": "^4.3.2"

--- a/package.json
+++ b/package.json
@@ -28,6 +28,7 @@
         "lint": "npx eslint src/**/*.ts"
     },
     "dependencies": {
+        "@textlint/ast-node-types": "^12.2.2",
         "@types/date-fns": "^2.6.0",
         "orga": "2.4.9",
         "structured-source": "^3.0.2",

